submissions/ksalpern/Hooli-style-Popup/scss/style.scss

```diff
@@ -0,0 +1,336 @@
+html {
+  box-sizing: border-box;
+  height: 100%;
+}
+
+*,
+*::after,
+*::before {
+  box-sizing: inherit;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  min-width: 320px;
+  overflow-x: hidden;
+  height: 100%;
+  font-family: Montserrat, sans-serif;
+}
+
+ol,
+ul {
+  list-style: none;
+}
+
+a,
+a:link,
+a:visited,
+a:hover {
+  text-decoration: none;
+}
+
+img {
+  height: auto;
+  vertical-align: top;
+}
+
+.header {
+  max-width: 1200px;
+  margin: 0 auto;
+  background-color: rgb(212, 240, 231);
+  padding: 24px 0px;
+}
+
+.header__nav {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+
+  .nav__items {
```

Comment:
Looks like you forget to fix nesting here)
For this time I merge this PR but in future avoid mistakes like this)

Reply:
Alright) Thanks for your time, I appreciate your efforts)